recipes/tensorflow/samples/tensorflow/retail/retail/tensorflow/scorer.py

```diff
@@ -0,0 +1,142 @@
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+
+from ml.runtime.python.Interfaces.AbstractScorer import AbstractScorer
+from data_access_sdk_python.reader import DataSetReader
+from data_access_sdk_python.writer import DataSetWriter
+import tensorflow as tf
+
+import pandas as pd
+fc = tf.feature_column
+tf.set_random_seed(123)
+
+import os
+import json
+
+class Scorer(AbstractScorer):
+    def score(self, config={}):
+        tf.logging.set_verbosity(tf.logging.ERROR)
+        tf_config = json.loads(os.environ['TF_CONFIG'])
+        tf_config = json.loads('{}')
+        os.environ['TF_CONFIG'] = json.dumps(tf_config)
+
+
+        #########################################
+        # Load Data
+        #########################################
+        prodreader = DataSetReader(client_id=config['ML_FRAMEWORK_IMS_USER_CLIENT_ID'],
+                                   user_token=config['ML_FRAMEWORK_IMS_TOKEN'],
+                                   service_token=config['ML_FRAMEWORK_IMS_ML_TOKEN'])
+
+        dataframe = prodreader.load(data_set_id=config['scoringDataSetId'],
+                                    ims_org=config['ML_FRAMEWORK_IMS_ORG_ID'])
+
+
+
+        #########################################
+        # Data Preparation/Feature Engineering
+        #########################################
+        if '_id' in dataframe.columns:
+            # Rename columns to strip tenantId
+            dataframe = dataframe.rename(columns=lambda x: str(x)[str(x).find('.') + 1:])
+            # Drop id, eventType and timestamp
+            dataframe.drop(['_id', 'eventType', 'timestamp'], axis=1, inplace=True)
+
+        dataframe.date = pd.to_datetime(dataframe.date)
+        dataframe['week'] = dataframe.date.dt.week
+        dataframe['year'] = dataframe.date.dt.year
+
+        dataframe = dataframe.sort_values(by=['date', 'store'])
+
+        dataframe = pd.concat([dataframe, pd.get_dummies(dataframe['storeType'])], axis=1)
+        dataframe.drop('storeType', axis=1, inplace=True)
+        dataframe['isHoliday'] = dataframe['isHoliday'].astype(int)
+
+        dataframe['weeklySalesAhead'] = dataframe.shift(-45)['weeklySales']
+        dataframe['weeklySalesLag'] = dataframe.shift(45)['weeklySales']
+        dataframe['weeklySalesDiff'] = (dataframe['weeklySales'] - dataframe['weeklySalesLag']) / dataframe[
+            'weeklySalesLag']
+        dataframe.dropna(0, inplace=True)
+
+        dataframe = dataframe.set_index(dataframe.date)
+        dataframe.drop('date', axis=1, inplace=True)
+
+
+
+        #########################################
+        # Data Preparation/Feature Engineering
+        #########################################
+        X_test = dataframe.drop('weeklySalesAhead', axis=1)
+        y_test = dataframe['weeklySalesAhead'].values
```

Comment:
Purposely lower case 'y' here?

Reply:
Yes, this is a common convention in these type of tutorials.